### ISSUE TITLE
[Talk TO Expert ✔️+(1)-803-993-9177] how To Cancel CVS carepass™ subscription

### ISSUE THREAD

**Issue by iamalessiatummino** [typescript-eslint/typescript-eslint#10404]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.
- [X] I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3A%22package%3A+eslint-plugin%22) and found none that matched my issue.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

### Playground Link

https://github.com/typescript-eslint/typescript-eslint/issues/10403

### Repro Code

```TypeScript
To cancel your Plenty of Fish (POF) subscription, follow these steps:

    Open POF App or Website: Log in to your account.
    Access Account Settings: Go to the settings or "My Account" section.
    Subscription Management: Look for the "Subscription" or "Payment" option.
    Cancel Subscription: Select the option to cancel your subscription. Follow the prompts to confirm the cancellation.
    Check Confirmation: You should receive a confirmation email. Keep it for reference.

For mobile devices, if you subscribed through iOS or Android, manage your subscription through the App Store or Google Play Store settings or talk to +(1)-803-993-9177.
```

### ESLint Config

```javascript
module.exports = {
  parser: "@typescript-eslint/parser",
  rules: {
    "@typescript-eslint/<rule-name>": ["error", ...<options>],
  },
};
```

### tsconfig

```jsonc
{
  "compilerOptions": {
    // ...
  }
}
```

### Expected Result

[Talk TO Expert ✔️+(1)-803-993-9177] how To Cancel CVS carepass™ subscription

### Actual Result

[Talk TO Expert ✔️+(1)-803-993-9177] how To Cancel CVS carepass™ subscription
